app/models/blood_pressure.rb

```diff
@@ -0,0 +1,17 @@
+class BloodPressure < ApplicationRecord
+  include Mergeable
+  belongs_to :patient, optional: true
+
+  def with_payload_keys(attributes)
```

Comment:
Seems like we could turn this into a model concern if we're copying this code into multiple models.